style.css

```diff
@@ -522,6 +533,53 @@ textarea::placeholder {
   gap: 16px;
 }
 
+.project-details {
+  min-height: 600px;
+  background-color: rgba(0, 0, 0, 0.527);
+  position: fixed;
+  z-index: 10;
+  top: 0;
+  width: 100%;
+}
+
+.modal {
+  margin: 8px 24px;
+  padding: 20px 24px;
+  background: white;
+  overflow: auto;
+  height: 600px;
+}
+
+.modal-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-family: 'Roboto', Courier, monospace;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  color: #172b4d;
+}
+
+.tech-details li {
+  margin-bottom: 10px;
+}
+
+.descript {
+  font-family: 'Inter', Courier, monospace;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.pro-links {
+  display: flex;
+  justify-content: space-between;
+  gap: 20px;
+}
```

Comment:
- [ ] For this project, your design should look exactly like the design provided by the Figma template. Kindly, fix your buttons' link margin so that your design looks like the Figma design.

|Your Design|Figma Design|
| ------------ |---------------|
|![image](https://user-images.githubusercontent.com/71874425/174270884-633be8d3-785c-4aeb-ab17-ad67922c9865.png)|![image](https://user-images.githubusercontent.com/71874425/174269828-97baf1a3-789d-4e99-b62b-3aee74e14566.png)|